apps/right-to-rent-check/acceptance/features/tenant-additional-details.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const _ = require('lodash');
+const steps = require('../../');
+
+Feature('Given I am at /tenant-additional-details');
```

Comment:
the urls' are canonical. However, things we call pages changes and is different from person to person. In my view urls are more reliable. It is not bad